selfdrive/car/hyundai/carstate.py

```diff
@@ -88,7 +88,7 @@ def update(self, cp, cp_cam):
         ret.gearShifter = GearShifter.unknown
     # Gear Selecton - This is only compatible with optima hybrid 2017
     elif self.CP.carFingerprint in FEATURES["use_elect_gears"]:
```

Comment:
Update `use_elect_gears` to match the new signal name

Reply:
i did not understand? you want to me change name from use_elect_gears to use_ev_pc5 ?

Reply:
yeah, something like `use_ev_gears`